Jenkinsfile

```diff
@@ -584,7 +584,7 @@ EOF
                                 export NO_PROXY
                             fi
 
-                            if ! time mvn -pl integration-tests -P ${MAVEN_PROFILE_NAME} verify 2>&1 | tee "${result_root}/kindtest.log"; then
+                            if ! time mvn -pl integration-tests -Dgroups="gate,samples" -DskipITs=false verify 2>&1 | tee "${result_root}/kindtest.log"; then
```

Comment:
I thought we were going to change the Maven profiles to specify the groups instead of the test classes?

Reply:
For that I would have to tag all the classes and change all the profiles to use the groups. @anpanigr suggested to first do the `gate` and then go for the rest.

Reply:
Right, but now you have a Jenkinsfile job with a dead parameter (MAVEN_PROFILE_NAME) that isn't being used anywhere.

Reply:
If we just changed the `wls-srg` profile to use groups, wouldn't that allow you to do just the gate/samples-gate tags first and not have to convert everything else?

Reply:
I think I tried that and did not work. Let me try again and see.

Reply:
modified `wls-srg` profile to use groups